### PR TITLE
{Release} Hotfix: Release 2.33.1

### PR DESCRIPTION
**Pull request by zhoxing-ms**
**Description**

Related PR
1. https://github.com/Azure/azure-cli/pull/21241

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change.
[Component Name 2] `az command b`: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Review by kairu-ms — APPROVED** [review]

**Review by wangzelin007 — APPROVED** [review]

**Review by evelyn-ys — APPROVED** [review]

**Review by jiasli — APPROVED** [review]

**Review by calvinhzy — APPROVED** [review]

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,17 @@
 Release History
 ===============
 
+2.33.1
+++++++
+
+**Compute**
+
+* Hotfix: Fix #21224: Fix the issue that VMCustomization is not enabled (#21241)
```
Comment (jsntcy) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* Hotfix: Fix the issue that VMCustomization is not enabled (#21241)
```

**Comment by jiasli** [comment]
Linking to #21277 which is required to fix the CI pipeline.

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (jiasli) [reply src/azure-cli/HISTORY.rst:11]: It should be ok to include issue ID that is fixed. The issue ID is useful especially when someone is tracking this issue.

Comment (jsntcy) [inline src/azure-cli/HISTORY.rst:11]:
```suggestion
* [Hotfix] Fix #21224: Fix the issue that VMCustomization is not enabled (#21241)
```

File: src/azure-cli/HISTORY.rst [hunk 2]
```diff
@@ -3,6 +3,17 @@
...
+
+**Packaging**
+
+* BREAKING CHANGE: Drop jmespath-terminal from docker image (#21277)
```
Comment (jsntcy) [inline src/azure-cli/HISTORY.rst:15]:
```suggestion
* [BREAKING CHANGE]: Drop jmespath-terminal from docker image (#21277)
```

File: src/azure-cli/HISTORY.rst [hunk 3]
```diff
@@ -3,6 +3,17 @@
...
+2.33.1
+++++++
+
+**Compute**
+
+* Hotfix: Fix #21224: Fix the issue that VMCustomization is not enabled (#21241)
+
+**Packaging**
+
+* [BREAKING CHANGE]: Drop jmespath-terminal from docker image (#21277)
```
Comment (zhoxing-ms) [inline src/azure-cli/HISTORY.rst:15]:
```suggestion
* [BREAKING CHANGE] Drop jmespath-terminal from docker image (#21277)
```

**Review by jsntcy — APPROVED** [review]

**Pull request merged**